### PR TITLE
chore: output hostname in index.html

### PR DESCRIPTION
**Pull request by js-315385995**

### CODE REVIEW

File: beginner/webserver/index.html [hunk 1]
```diff
@@ -1,31 +1,4 @@
 <!DOCTYPE html>
-
-<head>
-	<title>Docker Example</title>
-	<style>
```
Comment (the-technat) [inline beginner/webserver/index.html:5]: Can we add the styling again? Or why did you remove this?

File: beginner/webserver/index.html [hunk 2]
```diff
@@ -1,31 +1,4 @@
...
-
-<body>
-	<h1>This is a simple Webserver</h1>
-	<p>Yeah my Docker Image is working! Great work </p>
-</body>
-
-</html>
\ No newline at end of file
+<html>
+<h1>Hello from <!--#echo var="SERVER_NAME"--></h1>
```
Comment (asdfgugus) [inline beginner/webserver/index.html]: I built the docker image with this html content, but it seems not to return the server name. Does it work in your environment? Am I missing something?

File: beginner/webserver/index.html (hunk 1 above)
Comment (asdfgugus) [reply beginner/webserver/index.html:5]: I agree with @the-technat. It would be nice to use the current styling.

**Review by asdfgugus — CHANGES_REQUESTED** [review]

**Review by the-technat — APPROVED** [review]

File: beginner/webserver/index.html (hunk 2 above)
Comment (the-technat) [reply beginner/webserver/index.html]: Are you trying to do this with PHP?

Comment (asdfgugus) [reply beginner/webserver/index.html]: Found the issue! ssi on; needs to be configured directly under server{}.

Comment (asdfgugus) [reply beginner/webserver/index.html]: Fix from @js-315385995 works as well!

**Review by asdfgugus — APPROVED** [review]
